File: GTTS/INL.TrainingService.Models/ITrainingEventRoster_Item.cs
using System;
using System.Collections.Generic;

namespace INL.TrainingService.Models
{
    public interface ITrainingEventRoster_Item
    {
        long? TrainingEventRosterID { get; set; }
        long TrainingEventID { get; set; }
        long PersonID { get; set; }
        string FirstMiddleNames { get; set; }
        string LastNames { get; set; }
        byte? PreTestScore { get; set; }
        byte? PostTestScore { get; set; }
        byte? PerformanceScore { get; set; }
        byte? ProductsScore { get; set; }
        byte? AttendanceScore { get; set; }
        byte? FinalGradeScore { get; set; }
        bool? Certificate { get; set; }
        bool? MinimumAttendanceMet { get; set; }
        int? TrainingEventRosterDistinctionID { get; set; }
        string TrainingEventRosterDistinction { get; set; }
        byte? NonAttendanceReasonID { get; set; }
        string NonAttendanceReason { get; set; }
        byte? NonAttendanceCauseID { get; set; }
        string NonAttendanceCause { get; set; }
        string Comments { get; set; }
        int ModifiedByAppUserID { get; set; }
        DateTime ModifiedDate { get; set; }

        List<TrainingEventAttendance_Item> Attendance { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/itraining-event-roster_item.ts



import { TrainingEventAttendance_Item } from './training-event-attendance_item';

export interface ITrainingEventRoster_Item {
  
	TrainingEventRosterID?: number;
	TrainingEventID: number;
	PersonID: number;
	FirstMiddleNames: string;
	LastNames: string;
	PreTestScore?: number;
	PostTestScore?: number;
	PerformanceScore?: number;
	ProductsScore?: number;
	AttendanceScore?: number;
	FinalGradeScore?: number;
	Certificate?: boolean;
	MinimumAttendanceMet?: boolean;
	TrainingEventRosterDistinctionID?: number;
	TrainingEventRosterDistinction: string;
	NonAttendanceReasonID?: number;
	NonAttendanceReason: string;
	NonAttendanceCauseID?: number;
	NonAttendanceCause: string;
	Comments: string;
	ModifiedByAppUserID: number;
	ModifiedDate: Date;
	Attendance?: TrainingEventAttendance_Item[];
}





